src/lmnt/api.py

```diff
@@ -121,11 +123,13 @@ async def finish(self):
 
 
 class Speech:
-  def __init__(self, api_key: str, **kwargs):
+  def __init__(self, api_key: Optional[str] = None, **kwargs):
     self._session = None
-    self._api_key = api_key
+    self._api_key = api_key or os.environ.get('LMNT_API_KEY')
     self._base_url = kwargs.get('base_url', _BASE_URL)
     self._connector = kwargs.get('connector', None)
+    if self._api_key is None:
```

Comment:
I think an empty string will pass this -- perhaps `if not self._api_key:` instead?

Reply:
Done.